src/targets/__tests__/maven.test.ts

```diff
@@ -457,7 +457,9 @@ describe('closeRepository', () => {
 
     const mvnTarget = createMavenTarget();
     await expect(mvnTarget.closeRepository(repositoryId)).rejects.toThrow(
-      new Error('Unable to close repository: 500, Internal Server Error')
+      new Error(
+        'Unable to close repository sentry-java: 500, Internal Server Error'
```

Comment:
The repository name must be taken from the config. `sentry-java` is not the only repo using this target (see [`sentry-android-gradle-plugin`](https://github.com/getsentry/sentry-android-gradle-plugin/blob/464f228a42116ee995bda14e6fdae05398ec3725/.craft.yml#L4)), and there may be more in the future.

Reply:
It's take from the registry itself, those are just tests